examples/optional-components/deploy/example-production.horizontal_pod_autoscaler.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: example
+  namespace: staging
+spec:
+  minReplicas: 2
+  maxReplicas: 10
+  scaleTargetRef:
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+    name: example
+  targetCPUUtilizationPercentage: 75
+
```

Comment:
Unneeded newline. (pointing out as you awesomely remove them from other files!)

Reply:
Heh, my editor does, I can't take credit. Didn't get this one since I didn't edit it, just cp :). Fixed in next.